runtime/lua/vim/diagnostic.lua

```diff
@@ -152,6 +152,8 @@ local M = {}
 --- @field suffix? string|table|(fun(diagnostic:vim.Diagnostic,i:integer,total:integer): string, string)
 ---
 --- @field focus_id? string
+---
+--- @field border? string
```

Comment:
is there a generic float class that `vim.diagnostic.Opts.Float` can inherit from?

Reply:
just search float in help doc seems like we don't have a float class  ?

Reply:
yeah i don't see one in `runtime/lua/vim/_meta/api.lua`